### ISSUE TITLE
DatePicker component is not accessible

### ISSUE THREAD

**Issue by matuzalemsteles** [liferay/clay#5201]
DatePicker component is not accessible following w3c recommendations https://www.w3.org/WAI/ARIA/apg/example-index/dialog-modal/datepicker-dialog.

Both the ARIA semantic part for the elements and the standard keyboard navigation part in the grid do not exist in the component.

**Comment by matuzalemsteles** [comment]
cc @marcoscv-work

**Comment by matuzalemsteles** [comment]
A part of this issue has been merged but still needs to add keyboard interactions, I'll leave it to work on that next week.
